homeassistant/components/nice_go/coordinator.py

```diff
@@ -221,3 +227,22 @@ async def on_connected(self) -> None:
         """Handle the websocket connection."""
         _LOGGER.debug("Connected to the websocket")
         await self.api.subscribe(self.organization_id)
+        self.async_set_updated_data(self.data)
+
+    async def on_connection_lost(self, data: dict[str, Exception]) -> None:
+        """Handle the websocket connection loss. Don't need to do much since the library will automatically reconnect."""
+        _LOGGER.debug("Connection lost to the websocket")
+        self.async_set_update_error(data["exception"])
+
+    def unsubscribe(self) -> None:
+        """Unsubscribe from the websocket."""
+        if (
+            self._unsub_connected is None
```

Comment:
Given that you check for `None`, should you set these to `None` after calling them?